pandas/_libs/tslibs/parsing.pyx

```diff
@@ -44,13 +50,73 @@ class DateParseError(ValueError):
 
 _DEFAULT_DATETIME = datetime(1, 1, 1).replace(hour=0, minute=0,
                                               second=0, microsecond=0)
+_DEFAULT_TZINFO = _DEFAULT_DATETIME.tzinfo
 
 cdef:
     object _TIMEPAT = re.compile(r'^([01]?[0-9]|2[0-3]):([0-5][0-9])')
 
     set _not_datelike_strings = {'a', 'A', 'm', 'M', 'p', 'P', 't', 'T'}
 
 # ----------------------------------------------------------------------
+DEF delimiters = b' /-\\'
+DEF MAX_DAYS_IN_MONTH = 31
+DEF MAX_MONTH = 12
+
+cdef bint _is_not_delimiter(const char ch):
```

Comment:
we can't use flake8 on these files so have to be careful manually.  Two lines before/after a function or class